src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import core.basesyntax.application.FruitShopApplication;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operations.BalanceOperation;
+import core.basesyntax.operations.OperationHandler;
+import core.basesyntax.operations.OperationStrategy;
+import core.basesyntax.operations.OperationStrategyImpl;
+import core.basesyntax.operations.PurchaseOperation;
+import core.basesyntax.operations.ReturnOperation;
+import core.basesyntax.operations.SupplyOperation;
+import core.basesyntax.report.ReportGeneratorImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.services.DataConverterImpl;
+import core.basesyntax.services.FileReaderImpl;
+import core.basesyntax.services.FileWriterImpl;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+    private static final String DEFAULT_INPUT_FILE_PATH = "src/main/resources/reportToRead.csv";
+    private static final String DEFAULT_OUTPUT_FILE_PATH = "src/main/resources/finalReport.csv";

```

Comment:
Avoid hardcoding file paths in your application. Instead, pass these as parameters to your application, or use a configuration file or environment variables.